pavelib/prereqs.py

```diff
@@ -255,3 +255,4 @@ def install_prereqs():
     install_node_prereqs()
     uninstall_python_packages()
     install_python_prereqs()
+    sh("pip uninstall -y --disable-pip-version-check south")
```

Comment:
@nedbat south was pulled in by open-id-auth, we have removed the south from that repo hence removed this line.